.chloggen/fix_red-metrics.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fix RED metric retrieval from prometheus by removing the in Jaeger 1.62 introducted default RED metric namespace.
```

Comment:
`Remove the default metrics namespace/prefix for span RED metrics in Jaeger query`